benches/dapol.rs

```diff
@@ -76,9 +76,7 @@ fn bench_build_tree(c: &mut Criterion) {
                     format!("{:?}/{:?}/NUM_USERS: {:?}", &tup.0, &tup.1, &tup.2),
                 ),
                 |bench| {
-                    bench.iter(|| {
-                        ndm_smt = setup::build_ndm_smt(tup.clone());
-                    })
+                    bench.iter(|| ndm_smt = setup::build_ndm_smt(tup.clone()).unwrap());
```

Comment:
This is just moving the panic around. Same question: 

> If a bench panics then does the whole process die or does it move onto the next bench?

Reply:
It should probably move on and give a message